src/styles/card.scss

```diff
@@ -0,0 +1,121 @@
+.card {
+  margin: auto;
+  width: 200px;
```

Comment:
Would be great to create a variable for this value

Reply:
@denzzlle171 check this comment please